Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHG_cff.py

```diff
@@ -0,0 +1,77 @@
+from RecoVertex.BeamSpotProducer.BeamSpot_cfi import offlineBeamSpot
+
+# Ingredient: AlignmentTrackSelector
+# track selector for HighPurity tracks
+#-- AlignmentTrackSelector
+SiPixelAliHighPuritySelectorHG = SiPixelAliHighPuritySelector.clone()
+
+# track selection for alignment
+SiPixelAliTrackSelectorHG = SiPixelAliTrackSelector.clone(
+	src = 'SiPixelAliTrackFitterHG'
+)
+
+# Ingredient: SiPixelAliTrackRefitter0
+# refitting
+SiPixelAliTrackRefitterHG0 = SiPixelAliTrackRefitter0.clone(
+	src = 'SiPixelAliHighPuritySelectorHG'
+)
+SiPixelAliTrackRefitterHG1 = SiPixelAliTrackRefitterHG0.clone(
+	src = 'SiPixelAliTrackSelectorHG'
+)
+
+#-- Alignment producer
+SiPixelAliMilleAlignmentProducerHG = SiPixelAliMilleAlignmentProducer.clone()
+SiPixelAliMilleAlignmentProducerHG.ParameterBuilder.Selector = cms.PSet(
+    alignParams = cms.vstring(
+	"TrackerP1PXBLadder,111111",
+	"TrackerP1PXECPanel,111111",
+        )
+    )
+SiPixelAliMilleAlignmentProducerHG.tjTkAssociationMapTag = 'SiPixelAliTrackRefitterHG1'
+SiPixelAliMilleAlignmentProducerHG.algoConfig = MillePedeAlignmentAlgorithm.clone()
+SiPixelAliMilleAlignmentProducerHG.algoConfig.binaryFile = 'milleBinaryHG_0.dat'
+SiPixelAliMilleAlignmentProducerHG.algoConfig.treeFile = 'treeFileHG.root'
+SiPixelAliMilleAlignmentProducerHG.algoConfig.monitorFile = 'millePedeMonitorHG.root'
+
+# Ingredient: SiPixelAliTrackerTrackHitFilter
+SiPixelAliTrackerTrackHitFilterHG = SiPixelAliTrackerTrackHitFilter.clone(
+	src = 'SiPixelAliTrackRefitterHG0'
+)
+
+# Ingredient: SiPixelAliSiPixelAliTrackFitter
+SiPixelAliTrackFitterHG = SiPixelAliTrackFitter.clone(
+	src = 'SiPixelAliTrackerTrackHitFilterHG'
+)
+
+SiPixelAliMillePedeFileConverterHG = cms.EDProducer("MillePedeFileConverter",
+                                                  #FIXME: convert to untracked?
+                                                  fileDir = cms.string(SiPixelAliMilleAlignmentProducerHG.algoConfig.fileDir.value()),
+                                                  inputBinaryFile = cms.string(SiPixelAliMilleAlignmentProducerHG.algoConfig.binaryFile.value()),
+                                                  #FIXME: why was the label removed? Don't we want a label?
+                                                  fileBlobLabel = cms.string(''),
+                                                 )
+
+
+
+seqALCARECOPromptCalibProdSiPixelAliHG = cms.Sequence(ALCARECOTkAlMinBiasFilterForSiPixelAliHG*
+                                                    lsNumberFilter*
+                                                    offlineBeamSpot*
+                                                    SiPixelAliHighPuritySelectorHG*
+                                                    SiPixelAliTrackRefitterHG0*
+                                                    SiPixelAliTrackerTrackHitFilterHG*
+                                                    SiPixelAliTrackFitterHG*
+						    SiPixelAliTrackSelectorHG*
```

Comment:
I'm a bit confused, because I can't see the wrong indent in my code nor in https://github.com/cms-sw/cmssw/blob/be1db25175ed73809f6491d5373b5a7c6e170516/Alignment/CommonAlignmentProducer/python/ALCARECOPromptCalibProdSiPixelAliHG_cff.py#L74